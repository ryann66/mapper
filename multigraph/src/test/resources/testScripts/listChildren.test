CreateGraph g1
AddNode g1 alpha
AddNode g1 bravo
AddNode g1 charlie
ListChildren g1 alpha
AddEdge g1 alpha bravo kilo
ListChildren g1 alpha
#add to back
AddEdge g1 alpha charlie lima
ListChildren g1 alpha
#add to front
AddEdge g1 alpha alpha mike
ListChildren g1 alpha
#test secondary sort
AddEdge g1 alpha bravo november
ListChildren g1 alpha