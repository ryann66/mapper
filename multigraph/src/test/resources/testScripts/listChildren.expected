created multigraph g1
added node alpha to g1
added node bravo to g1
added node charlie to g1
the children of alpha in g1 are:
added edge kilo from alpha to bravo in g1
the children of alpha in g1 are: bravo(kilo)
#add to back
added edge lima from alpha to charlie in g1
the children of alpha in g1 are: bravo(kilo) charlie(lima)
#add to front
added edge mike from alpha to alpha in g1
the children of alpha in g1 are: alpha(mike) bravo(kilo) charlie(lima)
#test secondary sort
added edge november from alpha to bravo in g1
the children of alpha in g1 are: alpha(mike) bravo(kilo) bravo(november) charlie(lima)